sources/packages/api/src/route-controllers/application-exception/application-exception.aest.controller.ts

```diff
@@ -119,4 +125,32 @@ export class ApplicationExceptionAESTController extends BaseController {
       throw error;
     }
   }
+
+  /**
+   * Gets all student application exceptions.
+   * @param pagination options to execute the pagination.
+   * @returns list of student application exceptions.
+   */
+  @Get()
+  async getExceptions(
+    @Query() pagination: ApplicationStatusPaginationOptionsAPIInDTO,
+  ): Promise<PaginatedResultsAPIOutDTO<ApplicationExceptionSummaryAPIOutDTO>> {
+    const applicationExceptions =
+      await this.applicationExceptionService.getApplicationExceptions(
```

Comment:
Maybe the ticket is not clear or I am under the wrong impression, this query is supposed to display only "Pending" exceptions, right?

Reply:
Correct, missed it. I will update.